solr/core/src/java/org/apache/solr/handler/component/SearchHandler.java

```diff
@@ -622,8 +622,8 @@ public void handleRequestBody(SolrQueryRequest req, SolrQueryResponse rsp) throw
       } while (nextStage != Integer.MAX_VALUE);
 
       if (publishCpuTime) {
-        rsp.getResponseHeader().add(ThreadStats.CPU_TIME, totalShardCpuTime);
-        rsp.addToLog(ThreadStats.CPU_TIME, totalShardCpuTime);
+        rsp.getResponseHeader().add(ThreadCpuTime.CPU_TIME, totalShardCpuTime);
```

Comment:
No null check here?

Reply:
This can never be null due to the assignment that depends on the same condition in the if() around line 221.

Reply:
Do you mean for the responseHeader?